modules/tinymce/src/themes/silver/main/ts/ui/skin/Loader.ts

```diff
@@ -11,6 +11,22 @@ declare let tinymce: TinyMCE;
 import * as Options from '../../api/Options';
 import * as SkinLoaded from './SkinLoaded';
 
+const getSkinResourceIdentifier = (editor: Editor): Optional<string> => {
+  const oxideUiSkinMap: Record<string, string> = {
+    'oxide-dark': 'dark',
+    'oxide': 'default',
+    'tinymce-5': 'tinymce-5',
+    'tinymce-5-dark': 'tinymce-5-dark'
```

Comment:
Since these last two are the same, can we get rid of them?

Auto-translating skin `oxide-dark` to content `dark` seems like a new feature? I can't find any reference to it - all the docs say both settings are required. I like the idea, but maybe this is not the best time 🤔

Reply:
The issue is that when you specify `skin: 'oxide-dark'`, the `skin.js` file has `tinymce.Resource.add('ui/dark/skin.css'` so I needed to do a mapping to get the correct resource key value.

Now that I think about it, maybe that is where the fix should be made and the resource key should be  `tinymce.Resource.add('ui/oxide-dark/skin.css'` instead so a mapping isn't needed

Reply:
Oh! you've discovered a flaw in our theme build process. I knew I'd seen this mapping before. In the source folders, the skin is called `dark`.
https://github.com/tinymce/tinymce/blob/3eb6af8a9de284a67ec677fc825ef437bb6394f2/modules/oxide/src/less/skins/ui/dark/skin.less

We rename it in the tinymce bundle, but we don't replace the contents of the new JS files that are based on the source skin name.
https://github.com/tinymce/tinymce/blob/48e4df0806fd0ac73a88fffc4220b89f53712ba7/modules/tinymce/Gruntfile.js#L28-L33

We can either apply the rename to the content during the build, or rename the source skin to match the one we ship.

Reply:
I would be hesitant to change the source skin names at the 'oxide'module level given we rely on it in the premium skins.
I'll look at replacing the resource key it in the tinymce Gruntfile